src/applications/coronavirus-vaccination-expansion/config/military-history/helper.js

```diff
@@ -0,0 +1,32 @@
+const dischargeTypeLabels = {
+  honorable: 'Honorable',
+  general: 'General',
+  other: 'Other Than Honorable',
+  'bad-conduct': 'Bad Conduct',
+  dishonorable: 'Dishonorable',
+  undesirable: 'Undesirable',
+};
+
+const lastServiceBranchLabels = {
+  'air force': 'Air Force',
+  army: 'Army',
+  'coast guard': 'Coast Guard',
+  'marine corps': 'Marine Corps',
+  'merchant seaman': 'Merchant Seaman',
+  navy: 'Navy',
+  noaa: 'Noaa',
+  usphs: 'USPHS',
+  'f.commonwealth': 'Filipino Commonwealth Army',
+  'f.guerilla': 'Filipino Guerilla Forces',
+  'f.scouts new': 'Filipino New Scout',
+  'f.scouts old': 'Filipino Old Scout',
+  other: 'Other',
+};
```

Comment:
Can we make use of the platform const here:

https://github.com/department-of-veterans-affairs/vets-website/blob/0677518c2bd7a46e665cd1a54e3c32cb15a63a4e/src/platform/forms/address/data/labels.js#L6-L12

This seems even better:

https://github.com/department-of-veterans-affairs/vets-website/blob/0677518c2bd7a46e665cd1a54e3c32cb15a63a4e/src/platform/static-data/options-for-select.js#L9-L23